### PR TITLE
fix: windows CLI is installed in wrong location

### PR DESCRIPTION
**Pull request by benjaminSchilling33**
Before this commit, when installing the CLI using this script, it would move the downloaded CLI executable
as a file to `%LOCALAPPDATA%/Appwrite` while it is intended to move it to `%LOCALAPPDATA%/Appwrite/appwrite.exe`.

This commit fixes this problem by creating the directory explicitly and moving the file into it.

closes: https://github.com/appwrite/appwrite/issues/3007

### CODE REVIEW

**Comment by benjaminSchilling33** [comment]
@christyjacob4

**Comment by christyjacob4** [comment]
@TorstenDittmann  Can you give this a review ?

**Review by christyjacob4 — APPROVED** [review]
Template looks good to me. 
I'm not well versed into windows shell, so Im going to let @TorstenDittmann  help me there

Awesome work @benjaminSchilling33

**Comment by benjaminSchilling33** [comment]
In the meanwhile this seems to bei redundant to #397

**Comment by TorstenDittmann** [comment]
#397 actually also fixes different bugs, that are not covered in this PR.

For example running the installation twice will actually have an impact. So I think we might wanna merge #397 - which I think also comes wit the same fix from here.

File: templates/cli/install.ps1.twig [hunk 1]
```diff
@@ -16,11 +16,14 @@
 $GITHUB_x64_URL = "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName | caseDash }}/releases/download/{{ sdk.version }}/{{ language.params.executableName }}-cli-win-x64.exe"
 $GITHUB_arm64_URL = "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName | caseDash }}/releases/download/{{ sdk.version }}/{{ language.params.executableName }}-cli-win-arm64.exe"
 
+${{ spec.title | upper }}_BINRARY_NAME = "{{ language.params.executableName }}.exe"
```
Comment (christyjacob4) [inline templates/cli/install.ps1.twig:19]:
```suggestion
${{ spec.title | upper }}_BINARY_NAME = "{{ language.params.executableName }}.exe"
```

File: templates/cli/install.ps1.twig [hunk 2]
```diff
@@ -16,11 +16,14 @@
...
+
 # {{ spec.title | caseUcfirst }} download directory
-${{ spec.title | upper }}_DOWNLOAD_DIR = Join-Path -Path $env:TEMP -ChildPath "{{ language.params.executableName }}.exe"
+${{ spec.title | upper }}_DOWNLOAD_DIR = Join-Path -Path $env:TEMP -ChildPath ${{ spec.title | upper }}_BINRARY_NAME
```
Comment (christyjacob4) [inline templates/cli/install.ps1.twig:22]:
```suggestion
${{ spec.title | upper }}_DOWNLOAD_DIR = Join-Path -Path $env:TEMP -ChildPath ${{ spec.title | upper }}_BINARY_NAME
```

File: templates/cli/install.ps1.twig [hunk 3]
```diff
@@ -16,11 +16,14 @@
...
 
 # {{ spec.title | caseUcfirst }} CLI location
 ${{ spec.title | upper }}_INSTALL_DIR = Join-Path -Path $env:LOCALAPPDATA -ChildPath "{{ spec.title | caseUcfirst }}"
+${{ spec.title | upper }}_INSTALL_PATH = Join-Path -Path "${{ spec.title | upper }}_INSTALL_DIR" -ChildPath "${{ spec.title | upper }}_BINRARY_NAME"
```
Comment (christyjacob4) [inline templates/cli/install.ps1.twig:26]:
```suggestion
${{ spec.title | upper }}_INSTALL_PATH = Join-Path -Path "${{ spec.title | upper }}_INSTALL_DIR" -ChildPath "${{ spec.title | upper }}_BINARY_NAME"
```

**Comment by TorstenDittmann** [comment]
Thanks for the PR 🥳 

It will be part of the next release 🙂

**Pull request merged**